web/packages/shared/components/FieldSelect/FieldSelectCreatable.tsx

```diff
@@ -99,6 +106,8 @@ type CreatableProps = SelectCreatableProps & {
   autoFocus?: boolean;
   label?: string;
   rule?: (options: unknown) => () => unknown;
+  markAsError?: boolean;
```

Comment:
I'd add a JSDoc explaining what this prop is for. I assume it's to make it appear as if it was in an error state without using the `rule` prop, right?

Reply:
to me,  the name `markAsError` reads clear enough. Doesn't hurt to add JSDoc though https://github.com/gravitational/teleport/pull/36302/commits/6bca9b980c4f11543540bc197b8e148d285369a3